solr/src/main/resources/solr/conf/search-solr-managed-schema.xml

```diff
@@ -116,15 +116,15 @@
     <!-- docValues are enabled by default for long type so we don't need to index the version field  -->
     <field name="_version_" type="plong" indexed="false" stored="false"/>
 
-  <!-- CST fields -->
+    <!-- CST fields -->
   <field name="contents" type="text_general" multiValued="false" required="true" />
   <field name="country_code" type="string" />
   <field name="created" type="pdate" sortMissingLast="true" />
-  <field name="file_url" type="string" indexed="false" required="false" /> <!--TODO: Required? -->
+  <field name="file_url" type="string" required="false" indexed="false" /> <!--TODO: Required? -->
   <field name="icid" type="string" />
-  <field name="irm_url" type="string" indexed="false" required="true" />
+  <field name="irm_url" type="string" required="true" indexed="false" />
   <field name="keyword" type="string" />
-  <field name="metacard_url" type="string" indexed="false" required="false" /> <!--TODO: Required?-->
+  <field name="metacard_url" type="string" required="false" indexed="false" /> <!--TODO: Required?-->
```

Comment:
❓ Does stored default to true?

Reply:
Yes. Here is the reference I'm using. https://lucene.apache.org/solr/guide/8_2/field-type-definitions-and-properties.html#field-default-properties